presto-oracle/src/main/java/io/prestosql/plugin/oracle/OracleConfig.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.oracle;
+
+import io.airlift.configuration.Config;
+import io.prestosql.plugin.jdbc.BaseJdbcConfig;
+
+public class OracleConfig
+        extends BaseJdbcConfig
+{
+    private boolean includeSynonyms;
+    private int fetchSize = 1000;
+
+    public int getFetchSize()
+    {
+        return fetchSize;
+    }
+
+    @Config("oracle.fetch-size")
```

Comment:
Why does this need to be configurable?  I think it's should be OK to pick some good value (your default seems fine) and use this directly in the code.

Reply:
the more configurable the better, fetch size really depends on how narrow/wide the tables are

Reply:
@tooptoop4  good point
so it makes sense as a configuration when we can assume are tables are equal width.
which means we could calculate fetch size based on the number of columns being read

Reply:
removed